File: app/src/main/java/com/saphamrah/customer/data/local/db/entity/Bank.java
package com.saphamrah.customer.data.local.db.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "Bank")
public class Bank {

    public Bank(int ccBank, String bankTitle) {
        this.ccBank = ccBank;
        this.bankTitle = bankTitle;
    }

    @PrimaryKey
    @ColumnInfo(name = "ccBank")
    public int ccBank;


    @ColumnInfo(name = "bankTitle")
    public String bankTitle;

    /*@JvmField
    @ColumnInfo(name = "image", typeAffinity = ColumnInfo.BLOB)
    var image: ByteArray*/
}

File: app/src/main/java/com/saphamrah/customer/data/local/db/entity/Bank.kt
package com.saphamrah.customer.data.local.db.entity

import androidx.room.PrimaryKey
import androidx.room.ColumnInfo
import androidx.room.Entity

@Entity(tableName = "Bank")
data class Bank (
    @PrimaryKey
    @ColumnInfo(name = "ccBank")
    var ccBank: Int = 0,

    @JvmField
    @ColumnInfo(name = "bankTitle")
    var bankTitle: String? = null,

    /*@JvmField
    @ColumnInfo(name = "image", typeAffinity = ColumnInfo.BLOB)
    var image: ByteArray*/
)